src/iostream.c

```diff
@@ -684,10 +692,10 @@ static Int InitKernel(StructInitInfo * module)
     /* init filters and functions                                          */
     InitHdlrFuncsFromTable(GVarFuncs);
 
-#if !defined(HPCGAP)
-    /* Set up the trap to detect future dying children */
-    signal(SIGCHLD, ChildStatusChanged);
-#endif
+    if(!SIGCHLDHandlerDisabled) {
```

Comment:
Missing space after `if`

Reply:
Double negations are a potential cause of confusion, and so far, we only check `!SIGCHLDHandlerDisabled` -- so perhaps rename to `SIGCHLDHandlerEnabled`?